.travis.yml

```diff
@@ -109,22 +107,13 @@ before_install:
       }
       export -f tpecl
 
-    - |
-      # Install sigchild-enabled PHP to test the Process component on the lowest PHP matrix line
-      if [[ ! $deps && $TRAVIS_PHP_VERSION = ${MIN_PHP%.*} && ! -d php-$MIN_PHP/sapi ]]; then
-          wget http://php.net/get/php-$MIN_PHP.tar.bz2/from/this/mirror -O - | tar -xj &&
-          (cd php-$MIN_PHP && ./configure --enable-sigchild --enable-pcntl && make -j2)
-      fi
-
     - |
       # php.ini configuration
       (
-          for PHP in $TRAVIS_PHP_VERSION $php_extra; do
-              ([[ $PHP != 7.4 ]] && phpenv global $PHP 2>/dev/null) || (cd / && wget https://storage.googleapis.com/travis-ci-language-archives/php/binaries/ubuntu/18.04/x86_64/php-$PHP.tar.bz2 -O - | tar -xj) &
-          done
+          ([[ $TRAVIS_PHP_VERSION != 7.4 ]] && phpenv global $TRAVIS_PHP_VERSION 2>/dev/null) || (cd / && wget https://storage.googleapis.com/travis-ci-language-archives/php/binaries/ubuntu/18.04/x86_64/php-$TRAVIS_PHP_VERSION.tar.bz2 -O - | tar -xj) &
```

Comment:
do we still need this custom download ? We don't use multiple PHP versions anymore.